bld/namelist_files/namelist_defaults_ctsm_tools.xml

```diff
@@ -55,48 +55,19 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
    >lnd/clm2/mappingdata/grids/SCRIPgrid_0.1x0.1_nomask_c110712.nc</scripgriddata>
 <scripgriddata hgrid="0.5x0.5"   lmask="nomask"    
    >lnd/clm2/mappingdata/grids/SCRIPgrid_0.5x0.5_nomask_c110308.nc</scripgriddata>
-<scripgriddata hgrid="0.5x0.5"   lmask="AVHRR"     
-   >lnd/clm2/mappingdata/grids/SCRIPgrid_0.5x0.5_AVHRR_c110228.nc</scripgriddata>
-<scripgriddata hgrid="0.5x0.5"   lmask="MODIS"     
-   >lnd/clm2/mappingdata/grids/SCRIPgrid_0.5x0.5_MODIS_c110228.nc</scripgriddata>
-<scripgriddata hgrid="0.25x0.25" lmask="MODIS"     
-   >lnd/clm2/mappingdata/grids/SCRIPgrid_0.25x0.25_MODIS_c170321.nc</scripgriddata>
+<scripgriddata hgrid="0.25x0.25" lmask="nomask"     
+   >lnd/clm2/mappingdata/grids/SCRIPgrid_0.25x0.25_nomask_c191014.nc</scripgriddata>
 <scripgriddata hgrid="5x5min"    lmask="nomask"    
-   >lnd/clm2/mappingdata/grids/SCRIPgrid_5x5min_nomask_c110530.nc</scripgriddata>
-<scripgriddata hgrid="5x5min"    lmask="IGBP-GSDP" 
-   >lnd/clm2/mappingdata/grids/SCRIPgrid_5x5min_IGBP-GSDP_c110228.nc</scripgriddata>
-<scripgriddata hgrid="5x5min"   lmask="ISRIC-WISE"
-   >lnd/clm2/mappingdata/grids/SCRIPgrid_5x5min_ISRIC-WISE_c111114.nc</scripgriddata>
-<scripgriddata hgrid="5x5min"   lmask="ORNL-Soil"
-   >lnd/clm2/mappingdata/grids/SCRIPgrid_5x5min_ORNL-Soil_c170630.nc</scripgriddata>
+   >lnd/clm2/mappingdata/grids/SCRIPgrid_5x5min_nomask_fromIGBP_c191107.nc</scripgriddata>
```

Comment:
[Click to read comment about needing to replace this file with a generic SCRIPgrid_5x5min_nomask.nc file.](https://github.com/ESCOMP/CTSM/pull/823/files#r343967458)